pages/index.tsx

```diff
@@ -35,6 +37,7 @@ import TwitterIcon from "@material-ui/icons/Twitter";
 const StyledTabs = styled(Tabs)`
   & .MuiTabs-flexContainer {
     border-bottom: 1px solid #999;
+    width: 200%;
```

Comment:
this is added to ensure that the underline in the component extends far enough to accommodate many tabs in the case of including the analytics tab & weth.